Jellyfin.Data/Enums/MediaType.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace Jellyfin.Data.Enums;
+
+/// <summary>
+/// Media types.
+/// </summary>
+public enum MediaType
+{
+    /// <summary>
+    /// Unknown media type.
+    /// </summary>
+    Unknown = 0,
```

Comment:
Is it possible to get unknown media types via the API or do unknown media items only exist internally?

Reply:
I could probably go through and remove Unknown and have the default be null instead

Reply:
It's probably sane to have a good default value where everyone knows to just error out because something is wrong, right?